lint/linter/test-mdn-urls.ts

```diff
@@ -72,7 +109,25 @@ export const processData = (
         actual: data.mdn_url,
         expected: '',
       });
+    } else if (slugByPath.has(path) && !data.mdn_url?.includes('#')) {
+      // Overwrite url, unless it has a fragment.
+      const expected = `https://developer.mozilla.org/docs/${slugByPath.get(path)}`;
+      if (expected != data.mdn_url) {
+        issues.push({
+          ruleName: 'mdn_url_other_page',
+          path,
+          actual: data.mdn_url,
+          expected: `https://developer.mozilla.org/docs/${slugByPath.get(path)}`,
+        });
+      }
     }
+  } else if (slugByPath.has(path)) {
+    issues.push({
+      ruleName: 'mdn_url_new_page',
+      path,
+      actual: '',
+      expected: `https://developer.mozilla.org/docs/${slugByPath.get(path)}`,
+    });
   } else {
```

Comment:
I think this entire `else` branch that uses conventional places to get `mdn_urls` can now be removed. I tested this locally and your new heuristics using the front matter always proposed the correct mdn_urls for me. So, I think that long switch case becomes dead code and so we might as well remove it.

Reply:
Good point, fixed in [bc70078](https://github.com/mdn/browser-compat-data/pull/25563/commits/bc700783d41b2da2ff71fe8e67db5b19b2ca0880).